plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -19,9 +19,7 @@
 
 define('PLN_PLUGIN_NAME','plnplugin');
 
-define('PLN_PLUGIN_NETWORKS', serialize(array(
-	'PKP' => 'pkp-pln.lib.sfu.ca'
-)));
+define('PLN_PLUGIN_NETWORK', 'pkp-pln.lib.sfu.ca');
```

Comment:
(I have no idea what your arrangement with Mark is, BTW, so if I'm going beyond that, let me know.)